syntax="proto3";

package transport_proto;

message Edge {
	uint32 from = 1;
	uint32 to = 2;
	double weight = 3;
}

message IncidenceList {
	repeated uint32 incidence_list = 1;
}

message Graph {
	repeated Edge edges = 1;
	repeated IncidenceList incidence_lists = 2;
}